Sources/StreamVideoSwiftUI/CallView/ScreenSharing/ScreenSharingView.swift

```diff
@@ -45,44 +46,19 @@ public struct ScreenSharingView<Factory: ViewFactory>: View {
                     screensharingView.accessibility(identifier: "screenSharingView")
                 }
             }
-            
+
             if !viewModel.hideUIElements {
-                ScrollView(.horizontal) {
-                    HorizontalContainer {
-                        ForEach(viewModel.participants) { participant in
-                            viewFactory.makeVideoParticipantView(
-                                participant: participant,
-                                id: "\(participant.id)-screenshare-participant",
```

Comment:
I don't see this id persisted in the new code, any specific reason?

Reply:
This one is the track for the thumbnail. What's the difference between using this one or simply the participant track we already have (similarly on how we are handling it in Spotlight)?